src/react-apps/applications/shared/src/components/AltinnInputField.tsx

```diff
@@ -83,7 +85,10 @@ class AltinnInputField extends React.Component<IAltinnInputFieldComponentProvide
           </Typography>
         }
         {this.props.inputDescription &&
-          <Typography className={classNames(classes.descriptionInput, { [classes.marginTop_10]: this.props.inputHeader })}>
+          <Typography
+            style={this.props.inputDescriptionStyling}
```

Comment:
Have you checked that this does not break existing usage of this shared component?

Reply:
Yes, as null values does not alter the style this does not introduce any changes to components that does not explicity pass a inputDescriptionStyle.